### PR TITLE
feat(dashmate): add cli command for core service

### PR DESCRIPTION
**Pull request by pshenmic**
## Issue being fixed or feature implemented

Add a dashmate command to execute dash-cli commands in the core container. This is very useful, when you need to do something or check something with the dash-cli of the current node, but you have to use `docker exec -it..` instead.

## What was done?

* Added a cli command in the core folder

## How Has This Been Tested?

Locally

## Breaking Changes

No

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have added "!" to the title and described breaking changes in the corresponding section if my code contains any
- [x] I have made corresponding changes to the documentation if needed

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/dashmate/src/commands/core/cli.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+   * @return {Promise<void>}
+   */
+  async runWithDependencies(
+    args,
+    flags,
+    config,
+    dockerCompose,
+  ) {
+    const { command } = args;
+
```
Comment (shumkov) [inline packages/dashmate/src/commands/core/cli.js:19]: Let's add a check is service running?

File: packages/dashmate/src/commands/core/cli.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+const ConfigBaseCommand = require('../../oclif/command/ConfigBaseCommand');
+
+class DashCliCommand extends ConfigBaseCommand {
```
Comment (shumkov) [inline packages/dashmate/src/commands/core/cli.js:3]: CliCommand

**Review by shumkov — CHANGES_REQUESTED** [review]

**Comment by pshenmic** [comment]
Tested locally

With started testnet fullnode:
```
pshenmic@Mikhails-MacBook-Pro dashmate % yarn dashmate core cli "getblockhash 1"
0000047d24635e347be3aaaeb66c26be94901a2f962feccd4f95090191f208c1
```

With stopped testnet fullnode

```
pshenmic@Mikhails-MacBook-Pro dashmate % yarn dashmate core cli "getblockhash 1"
    ServiceIsNotRunningError: Service core for testnet is not running. Please run the service first.
```

**Review by shumkov — APPROVED** [review]
Nice 👍

**Pull request merged**